\newglossaryentry{abortive infection (AI)}{
	name={abortive infection (AI)},
	description={AI is a strategy by which bacteria block the propagation of phage by committing suicide upon infection.   Usually the AI gene or genes are encoded on a prophage but, unlike other prophage genes, are constitutively expressed.  AI generally is manifested by a sudden lethal membrane event at some time after the beginning of infection of a virulent phage, before the accumulation of progeny.  The molecular basis of AI events is unknown.   The best studied AI system is the  rex system encoded by phage lambda. }
}
\newglossaryentry{activator (transcription)}{
	name={activator (transcription)},
	description={Also transcription factor.  A protein that causes the activation of a cryptic promoter, either by binding to the DNA or to RNA polymerase. }
}
\newglossaryentry{annotation}{
	name={annotation},
	description={Annotation is the process of describing the contents of a genome.  M. Zorn has a nice definition:  Extraction, definition, and interpretation of features on the genome sequence derived by integrating computational tools and biological knowledge.  ().  The biggest task is finding the genes and assigning functional identities to the predicted gene products.}
}
\newglossaryentry{antiterminator}{
	name={antiterminator},
	description={A factor that reduces or abolishes transcriptional termination.    The first examples were the N and Q proteins of phage lambda.  However, in some cases, the anti-termination factor is a cis-acting RNA sequence.  See PMID:    .   }
}
\newglossaryentry{att}{
	name={att},
	description={Also attB, attP, attachment sites.  DNA sites on the phage DNA (attP) or bacterial chromosome (attB) that are used as substrates by phage-encoded Integrase for site-specific recombination.  Result is integration of the phage DNA into the host chromosome.}
}
\newglossaryentry{binary}{
	name={binary},
	description={knowledge.  ().  The biggest task is finding the genes and assigning functional identities to the predicted gene products.}
}
\newglossaryentry{burst size}{
	name={burst size},
	description={Average pfu per infective center; usually determined by single-step growth experiment}
}
\newglossaryentry{bushing}{
	name={bushing},
	description={A term applied to a 10-membered ring of the lambda MCP, gpE,  in contact with the portal.   These E proteins are modified by covalent exchange reactions with C proteins, generating  X1 and X2 products.  The bushing concept reflects some expectation that the portal ring may turn with respect to the capsid, as a consequence between the symmetry mismatch at the portal vertex (5 fold axis at each vertex but 12-fold rotational symmetry for the portal.)}
}
\newglossaryentry{Caudovirales}{
	name={Caudovirales},
	description={Taxon of the tailed phages (Myophages, Siphophages, Podophages)}
}
\newglossaryentry{cleared lysate}{
	name={cleared lysate},
	description={Crude product of phage infection or induction; cleared of large cell debris and usually stored over chloroform in the cold.}
}
\newglossaryentry{clear plaques}{
	name={clear plaques},
	description={hallmark of virulent phages; plaques lack visible  bacterial growth}
}
\newglossaryentry{coliphage}{
	name={coliphage},
	description={phage of E. coli}
}
\newglossaryentry{contractile tail}{
	name={contractile tail},
	description={tail of a myophage}
}
\newglossaryentry{cubic}{
	name={cubic},
	description={symmetry term that includes icosahedral symmetry}
}
\newglossaryentry{defective prophages}{
	name={defective prophages},
	description={prophages that have undergone mutations, including deletions, such that a viable virion is no longer encoded.  E. coli K-12 has several, including dlp at 12 min, for example.}
}
\newglossaryentry{E. coli B}{
	name={E. coli B},
	description={classic non-pathogenic laboratory strain of E. coli; source was J. Bordet.  Mandated for use with the T series (Seven Dwarves) in the Phage Course.}
}
\newglossaryentry{efficiency of plating}{
	name={efficiency of plating},
	description={usually the relative pfu titer on one strain compared to another.  Absolute EOP is pfu per virion.  This is ~1 for T4 and lambda.}
}
\newglossaryentry{Felix d'Herelle}{
	name={Felix d'Herelle},
	description={founder of phage biology.  See Module 1.}
}
\newglossaryentry{lysis}{
	name={Lysis},
	description={total clearing of the turbidity, associated with increased viscosity that we now know to result from the release of the bacterial DNA}
}

\newglossaryentry{adaptive immunity}{
    name={Adaptive Immunity},
    description={
    ``Adaptive immunity creates immunological memory after an initial response to a specific pathogen, leads to an enhanced response to subsequent encounters with that pathogen. This process of acquired immunity is the basis of vaccination. Like the innate system, the adaptive system includes both humoral immunity components and cell-mediated immunity components.''\cite{wiki:Adaptive_immune_system}
    }
}
\newglossaryentry{membrane potential}{
    name={Membrane Potential},
    description={}
}
\newglossaryentry{tape measure protein}{
    name={tape measure protein},
    description={}
}
\newglossaryentry{tail tube}{
    name={tail tube},
    description={}
}
\newglossaryentry{coiled coil}{
    name={coiled coil},
    description={}
}
\newglossaryentry{cohesive ends}{
    name={cohesive ends},
    description={}
}
\newglossaryentry{cos}{
    name={cos},
    description={}
}
\newglossaryentry{nicks}{
    name={nicks},
    description={}
}
\newglossaryentry{terminator}{
    name={terminator},
    description={}
}
\newglossaryentry{anti-terminator}{
    name={anti-terminator},
    description={}
}
\newglossaryentry{juggernaut}{
    name={juggernaut},
    description={}
}
\newglossaryentry{repressors}{
    name={repressors},
    description={}
}
\newglossaryentry{operator}{
    name={operator},
    description={}
}
\newglossaryentry{lysogenic repressor}{
    name={lysogenic repressor},
    description={}
}
\newglossaryentry{cryptic}{
    name={cryptic},
    description={}
}
\newglossaryentry{establishment}{
    name={establishment},
    description={}
}
\newglossaryentry{maintenance}{
    name={maintenance},
    description={}
}
\newglossaryentry{constitutive}{
    name={constitutive},
    description={}
}
\newglossaryentry{tyrosine recombinase}{
    name={tyrosine recombinase},
    description={}
}
\newglossaryentry{attP}{
    name={attP},
    description={}
}
\newglossaryentry{attB}{
    name={attB},
    description={}
}
\newglossaryentry{permuted}{
    name={permuted},
    description={}
}
\newglossaryentry{immunity}{
    name={immunity},
    description={}
}
\newglossaryentry{vir}{
    name={vir},
    description={}
}
\newglossaryentry{homo-immune}{
    name={homo-immune},
    description={}
}
\newglossaryentry{hetero-immune}{
    name={hetero-immune},
    description={}
}
\newglossaryentry{induction}{
    name={induction},
    description={}
}
\newglossaryentry{SOS response}{
    name={SOS response},
    description={}
}
\newglossaryentry{regulon}{
    name={regulon},
    description={}
}
\newglossaryentry{homolog}{
    name={homolog},
    description={}
}
\newglossaryentry{stochastic derepression}{
    name={stochastic derepression},
    description={}
}
\newglossaryentry{helix-turn-helix}{
    name={helix-turn-helix},
    description={}
}
\newglossaryentry{inverted repeat}{
    name={inverted repeat},
    description={}
}
\newglossaryentry{domains}{
    name={domains},
    description={}
}
\newglossaryentry{tripartite operators}{
    name={tripartite operators},
    description={}
}
\newglossaryentry{sub-operators}{
    name={sub-operators},
    description={}
}
