\newglossaryentry{abortive infection (AI)}{
	name={abortive infection (AI)},
	description={AI is a strategy by which bacteria block the propagation of phage by committing suicide upon infection.   Usually the AI gene or genes are encoded on a prophage but, unlike other prophage genes, are constitutively expressed.  AI generally is manifested by a sudden lethal membrane event at some time after the beginning of infection of a virulent phage, before the accumulation of progeny.  The molecular basis of AI events is unknown.   The best studied AI system is the  rex system encoded by phage lambda. }
}
\newglossaryentry{activator (transcription)}{
	name={activator (transcription)},
	description={Also transcription factor.  A protein that causes the activation of a cryptic promoter, either by binding to the DNA or to RNA polymerase. }
}
\newglossaryentry{annotation}{
	name={annotation},
	description={Annotation is the process of describing the contents of a genome.  M. Zorn has a nice definition:  Extraction, definition, and interpretation of features on the genome sequence derived by integrating computational tools and biological knowledge.  ().  The biggest task is finding the genes and assigning functional identities to the predicted gene products.}
}
\newglossaryentry{antiterminator}{
	name={antiterminator},
	description={A factor that reduces or abolishes transcriptional termination.    The first examples were the N and Q proteins of phage lambda.  However, in some cases, the anti-termination factor is a cis-acting RNA sequence.  See PMID:    .   }
}
\newglossaryentry{att}{
	name={att},
	description={Also attB, attP, attachment sites.  DNA sites on the phage DNA (attP) or bacterial chromosome (attB) that are used as substrates by phage-encoded Integrase for site-specific recombination.  Result is integration of the phage DNA into the host chromosome.}
}
\newglossaryentry{binary}{
	name={binary},
	description={knowledge.  ().  The biggest task is finding the genes and assigning functional identities to the predicted gene products.}
}
\newglossaryentry{burst size}{
	name={burst size},
	description={Average pfu per infective center; usually determined by single-step growth experiment}
}
\newglossaryentry{bushing}{
	name={bushing},
	description={A term applied to a 10-membered ring of the lambda MCP, gpE,  in contact with the portal.   These E proteins are modified by covalent exchange reactions with C proteins, generating  X1 and X2 products.  The bushing concept reflects some expectation that the portal ring may turn with respect to the capsid, as a consequence between the symmetry mismatch at the portal vertex (5 fold axis at each vertex but 12-fold rotational symmetry for the portal.)}
}
\newglossaryentry{Caudovirales}{
	name={Caudovirales},
	description={Taxon of the tailed phages (Myophages, Siphophages, Podophages)}
}
\newglossaryentry{cleared lysate}{
	name={cleared lysate},
	description={Crude product of phage infection or induction; cleared of large cell debris and usually stored over chloroform in the cold.}
}
\newglossaryentry{clear plaques}{
	name={clear plaques},
	description={hallmark of virulent phages; plaques lack visible  bacterial growth}
}
\newglossaryentry{coliphage}{
	name={coliphage},
	description={phage of E. coli}
}
\newglossaryentry{contractile tail}{
	name={contractile tail},
	description={tail of a myophage}
}
\newglossaryentry{cubic}{
	name={cubic},
	description={symmetry term that includes icosahedral symmetry}
}
\newglossaryentry{defective prophages}{
	name={defective prophages},
	description={prophages that have undergone mutations, including deletions, such that a viable virion is no longer encoded.  E. coli K-12 has several, including dlp at 12 min, for example.}
}
\newglossaryentry{E. coli B}{
	name={E. coli B},
	description={classic non-pathogenic laboratory strain of E. coli; source was J. Bordet.  Mandated for use with the T series (Seven Dwarves) in the Phage Course.}
}
\newglossaryentry{efficiency of plating}{
	name={efficiency of plating},
	description={usually the relative pfu titer on one strain compared to another.  Absolute EOP is pfu per virion.  This is \textasciitilde{}1 for T4 and lambda.}
}
\newglossaryentry{Felix d'Herelle}{
	name={Felix d'Herelle},
	description={founder of phage biology.  See Module 1.}
}
\newglossaryentry{lysis}{
	name={Lysis},
	description={total clearing of the turbidity, associated with increased viscosity that we now know to result from the release of the bacterial DNA}
}

\newglossaryentry{adaptive immunity}{
    name={Adaptive Immunity},
    description={
    ``Adaptive immunity creates immunological memory after an initial response to a specific pathogen, leads to an enhanced response to subsequent encounters with that pathogen. This process of acquired immunity is the basis of vaccination. Like the innate system, the adaptive system includes both humoral immunity components and cell-mediated immunity components.''\cite{wiki:Adaptive_immune_system}
    }
}
\newglossaryentry{membrane potential}{
    name={Membrane Potential},
    description={the voltage across the inner membrane of bacterial cells}
}
\newglossaryentry{tape measure protein}{
    name={tape measure protein},
    description={The tape measure protein determines the length of the phage tail}
}
\newglossaryentry{tail tube}{
    name={tail tube},
    description={a term describing the cylindrical tube located within the sheath; contains the tape measure protein}
}
\newglossaryentry{coiled coil}{
    name={coiled coil},
    description={a structural motif where multiple copies of a protein helix are coiled into a multimeric helix}
}
\newglossaryentry{cohesive ends}{
    name={cohesive ends},
    description={the region of unpaired nucleotides on each end of lambda DNA prior to circularization}
}
\newglossaryentry{cos}{
    name={cos},
    description={an abbreviation for the cohesive ends of lambda DNA}
}
\newglossaryentry{nicks}{
    name={nicks},
    description={refers to a small single-stranded region in dsDNA, usually only a few nucleotides}
}
\newglossaryentry{terminator}{
    name={terminator},
    description={a region of DNA sequence at the end of a gene which ends transcription}
}
\newglossaryentry{anti-terminator}{
    name={anti-terminator},
    description={a gene product which stops functionality of the terminator}
}
\newglossaryentry{juggernaut}{
    name={juggernaut},
    description={the juggernaut refers to the complex of N protein, RNA, and RNA polymerase. This complex does not allow termination as it moves forward transcribing RNA.}
}
\newglossaryentry{repressors}{
    name={repressors},
    description={a protein that binds to an operator and inhibits transcription of that gene}
}
\newglossaryentry{operator}{
    name={operator},
    description={a region of DNA that binds transcription factors and regulates gene expression}
}
\newglossaryentry{lysogenic repressor}{
    name={lysogenic repressor},
    description={an example in phage biology would be CI protein which, by repressing competing proteins, causes the pathway to proceed lysogenically}
}
\newglossaryentry{cryptic}{
    name={cryptic},
    description={a cryptic promoter has low or no functionality without transcription factors}
}
\newglossaryentry{establishment}{
    name={establishment},
    description={an established promoter is in its active form}
}
\newglossaryentry{constitutive}{
    name={constitutive},
    description={a constitutive promoter is consistently active in the cell}
}
\newglossaryentry{tyrosine recombinase}{
    name={tyrosine recombinase},
    description={the enzyme required for site-specific recombination of phage DNA}
}
\newglossaryentry{attP}{
    name={attP},
    description={a site on lambda DNA necessary for recombination}
}
\newglossaryentry{attB}{
    name={attB},
    description={a site of E. coli DNA necessary for recombination}
}
\newglossaryentry{permuted}{
    name={permuted},
    description={a phage genome where no defined start or stop is present}
}
\newglossaryentry{immunity}{
    name={immunity},
    description={immunity from lambda phage is acheived when a lambda lysogen is established}
}
\newglossaryentry{vir}{
    name={vir},
    description={refers to vir mutants}
}
\newglossaryentry{homo-immune}{
    name={homo-immune},
    description={homo-immune infection refers of a phage lysogen which is immune to infection by other phage DNA}
}
\newglossaryentry{hetero-immune}{
    name={hetero-immune},
    description={a hetero-immmune infection occurs when phage DNA successfully infects a lambda lysogen}
}
\newglossaryentry{induction}{
    name={induction},
    description={the process of phage DNA exiting the lysogenic state and undergoing the lytic cycle}
}
\newglossaryentry{SOS response}{
    name={SOS response},
    description={DNA damage results in this mechanism triggering a lysogen to undergo the lytic cycle}
}
\newglossaryentry{regulon}{
    name={regulon},
    description={a series of genes that are similarly regulated}
}
\newglossaryentry{homolog}{
    name={homolog},
    description={a homologous protein has similar DNA sequence to a protein in a different organism}
}
\newglossaryentry{helix-turn-helix}{
    name={helix-turn-helix},
    description={a protein structural motif}
}
\newglossaryentry{inverted repeat}{
    name={inverted repeat},
    description={an inverted DNA sequence found in operators}
}
\newglossaryentry{domains}{
    name={domains},
    description={a structural unit of a protein}
}
\newglossaryentry{tripartite operators}{
    name={tripartite operators},
    description={a tripartite operator consists of three sub-operators, each with different binding requirements}
}
\newglossaryentry{sub-operators}{
    name={sub-operators},
    description={an operator within a tripartite operator}
}
\newglossaryentry{turbidity}{
    name={turbidity},
    description={a measure of suspended cells in a lysate}
}
\newglossaryentry{h mutant}{
    name={h mutant},
    description={short for host range, a mutant capable of infecting bacteria species other than its normal host}
}
%Look over temperate again. May not need all words linked.
\newglossaryentry{temperate}{
    name={temperate},
    description={a phage capable of the lysogenic or lytic pathway}
}
\newglossaryentry{virulent}{
    name={virulent},
    description={a phage only capable of replicating using the lytic pathway}
}
\newglossaryentry{tail spike}{
    name={tail spike},
    description={the distal end of the flexible tail in lambda}
}
\newglossaryentry{prolate}{
    name={prolate},
    description={charactarizes elongation in the vertical direction}
}
\newglossaryentry{Wac}{
    name={Wac},
    description={the Wac protein forms the phage whiskers and collar structures}
}
\newglossaryentry{lysozyme}{
    name={lysozyme},
    description={enzyme which catalyzes degredation of the bacterial cell wall}
}
\newglossaryentry{beta-helix}{
    name={beta-helix},
	description={a protein structural motif}
}
\newglossaryentry{Alt}{
    name={Alt},
	description={the Alt protein is ejected by the phage into the cytoplasm, where it modifies host RNA polymerase}
}
\newglossaryentry{replisome}{
    name={replisome},
	description={the multi-protein complex that functions to replicate DNA}
}
\newglossaryentry{Okazaki fragment}{
    name={Okazaki fragment},
	description={the DNA fragments synthesized on the lagging strand during replication}
}
\newglossaryentry{Holliday junction}{
    name={Holliday junction},
	description={the interchange junction of DNA strands during recombination}
}
\newglossaryentry{capsomer}{
    name={capsomer},
	description={the structural unit of virus head capsid}
}
\newglossaryentry{chaperonin}{
    name={chaperonin},
	description={multi-meric complex that aids in protein folding}
}
\newglossaryentry{myophage}{
    name={myophage},
	description={a phage type charactarized by a contractile tail}
}
\newglossaryentry{siphophage}{
	name={siphophage},
	description={a phage type charactarized by a long, noncontractile tail}
}
\newglossaryentry{podophage}{
	name={podophage},
	description={a phage type charactarized by a short tail}
}
\newglossaryentry{endolysin}{
	name={endolysin},
	description={enzyme that cleaves host cell wall during the lytic cycle}
}
\newglossaryentry{holin}{
	name={holin},
	description={protein which creates holes in the IM, allowing endolysins to be released into the cytoplasm}
}
\newglossaryentry{spanin}{
	name={spanin},
	description={functions in aiding the OM disruption during lysis}
}
\newglossaryentry{pinholin}{
	name={pinholin},
	description={pinholins create small gaps in the host IM, similar to holins}
}
\newglossaryentry{prophage}{
	name={prophage},
	description={phage DNA incorporated into the host chromosome}
}
\newglossaryentry{vRNAP}{
	name={vRNAP},
	description={virion RNA polymerase}
}
\newglossaryentry{deletionogenic event}{
	name={deletionogenic event},
	description={loss of DNA material during replication}
}
\newglossaryentry{pseudogene}{
	name={pseudogene},
	description={a gene which no longer functions due to a mutation}
}
\newglossaryentry{moron}{
	name={moron},
	description={a DNA element that is found in a specific region on the chromosome, but not in all phage particles}
}
\newglossaryentry{insensitivity}{
	name={insensitivity},
	description={TODO: Dr. Young}
}
\newglossaryentry{specificity}{
	name={specificity},
	description={TODO: Dr. Young}
}
